docs/tutorials/helloworld.md

```diff
@@ -171,7 +171,8 @@ public class NetworkCommandLine : MonoBehaviour
 
 9. Paste the copied code into your code editor.
 1. Save your changes. Your script will reload in the Unity Editor.
-1. Back in the Editor, select **File** > **Build Settings** > **Player Settings...**. Beneath **Settings for PC, Mac, & Linux Standalone**, click **Resolution and Presentation** to open the section options.
+1. Back in the Editor, open **Edit** -> **Project Settings**
+1. Select the **Player tab**. then expand the **Resolution and Presentation** to open the section options.
```

Comment:
I would probable make this - expand Resolution and Presentation _or_ expand the Resolution and Presentation tab.